elfi/bo/acquisition.py

```diff
@@ -203,6 +203,37 @@ def acquire(self, n_values, pending_locations=None):
         return ret
 
 
+class RandomAcquisition(AcquisitionBase):
+    """Acquisition purely from priors. This can be useful if parameters
+    in certain regions are forbidden (i.e. their pdf is zero).
+
+    Parameters
+    ----------
+    prior_list : list of Prior objects
+
+    """
+
+    def __init__(self, prior_list, *args, **kwargs):
+        self.prior_list = prior_list
+        n_priors = len(prior_list)
+
+        # hacky...
+        class DummyModel(object):
+            pass
+        model = DummyModel()
```

Comment:
Can't the model be just `None` as this will likely be only used independently? (ie. this is not a mixin)